src/main/java/dev/compactmods/machines/config/ServerConfig.java

```diff
@@ -76,6 +78,10 @@ private static void generateConfig() {
                 .comment("Command level required for changing room upgrades.")
                 .defineInRange("upgrades", Commands.LEVEL_GAMEMASTERS, Commands.LEVEL_ALL, Commands.LEVEL_OWNERS);
 
+        ALLOWED_OUTSIDE_MACHINE = builder
+                .comment("If true it'll teleport player out of machine dimension if player is outside of the machine (default: false)")
+                .define("allowoutside", false);
+
```

Comment:
Needs moved to a new section, this is in the `commands` section. Also, prefer Java casing, `allowOutside`